File: app/src/main/java/com/moudle/app/LayerDrawableActivity.java
package com.moudle.app;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import android.util.Log;
import android.widget.ImageView;

/**
 * LayerDrawable        layer
 */
public class LayerDrawableActivity extends AppCompatActivity {

    private static final String TAG = "LayerDrawableActivity";

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_layer);
        ImageView iv = findViewById(R.id.iv_layer);

        Log.e(TAG, "LayerDrawableActivity:" + iv.toString());
    }
}

File: app/src/main/java/com/moudle/app/activity/LayerDrawableActivity.kt
package com.moudle.app.activity

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import com.moudle.app.databinding.ActivityLayerBinding

/**
 * LayerDrawable        layer
 */
class LayerDrawableActivity : AppCompatActivity() {
    private lateinit var layerBinding: ActivityLayerBinding

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        layerBinding = ActivityLayerBinding.inflate(layoutInflater)
        setContentView(layerBinding.root)

        Log.e(TAG, "LayerDrawableActivity:${layerBinding.ivLayer}")
    }

    companion object {
        private const val TAG = "LayerDrawableActivity"
    }
}